airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -82,6 +84,7 @@
     ),
     "latest_oss_catalog_gcs_file": gcs_file_blob.configured({"gcs_prefix": CATALOG_FOLDER, "gcs_filename": "oss_catalog.json"}),
     "latest_cloud_catalog_gcs_file": gcs_file_blob.configured({"gcs_prefix": CATALOG_FOLDER, "gcs_filename": "cloud_catalog.json"}),
+    "specs_secrets_mask": gcs_file_blob.configured({"gcs_prefix": CATALOG_FOLDER, "gcs_filename": "specs_secrets_mask.yaml"}),
```

Comment:
@bnchrch I did not upload the specs_secrets_mask to GCS. I'd love a inputs from your side how you want to orchestrate this kind of operation. We're not yet uploading registries to GCS so I did not find a example to copy 😄

Reply:
There is no story currently! But lets do it in this PR!

Heres an example of how we already write to the reports/ folder in the GCS bucket

https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/catalog_report.py#L125

Reply:
Also I think we dont have to have "specs_secrets_mask" as a resource here as were currently not triggering anything from the file itself at the moment

Reply:
Done in 150001e62a297312c531116076e5305315fd3cac